README.md

```diff
@@ -1,13 +1,68 @@
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+
+
+  <p align="center">
+
 ![Crate](https://raw.githubusercontent.com/atulmy/atulmy.github.io/master/images/crate/hero-with-link.png)
 
 # Crate 👕👖📦
+  <br/>
+  <!-- <a href="https://youtu.be/7aLGTn9YG6Q">Youtube Walk-through</a>
+    <br /> -->
+    <a href="https://github.com/Kathybui732/Crate"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/Kathybui732/Crate">View Demo</a>
+    ·
+    <a href="https://github.com/Kathybui732/Crate/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/Kathybui732/Crate/issues">Request Feature</a>
+  </p>
+</p>
+
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+* [Usage](#usage)
+* [License](#license)
+* [Contact](#contact)
+* [Acknowledgements](#acknowledgements)
+
+
+## About The Project
+
+This project explores an existing brownfield codebase. We will be using the open source crate project. Crate is a platform, similar to Stitch Fix, that allows users to setup monthly subscriptions to receive trendy clothes and accessories. We will be exploring the codebase in order to add test coverage, fix bugs, and extend the current functionality. We will not focus on mobile development. 
+
+Our learning goals were to: 
+
+- Learn and apply strategies for understanding how to analyze a larger, existing code base
+- Apply strategies for reading and evaluating documentation
+- Explore and implement new concepts, patterns, or libraries that have not been explicitly taught while at Turing
+- Practice an advanced, professional git workflow
+
+### Built With
 
-#### Get monthly subscription of trendy clothes and accessories.
 - **API** built with Node, GraphQL, Express, Sequelize (MySQL) and JWT Auth
-- **WebApp** built with React and Redux along with Server Side Rendering (SSR) / SEO friendly
-- **Mobile** (Android and iOS) Native App build with React Native
+- **WebApp** built with React and Redux along with Server Side Rendering (SSR) / SEO friendly. 
 - Written in ES6+ using Babel + Webpack
-- Designed using Adobe Experience Design. Preview it [here](https://xd.adobe.com/view/a662a49f-57e7-4ffd-91bd-080b150b0317/).
+
+## Developers
```

Comment:
Should we add this to the Contributors section below?

Reply:
Oh yeah we can actually delete that Adobe line actually. In this section, I was focusing only on what we touch.

Reply:
@Kathybui732